src/worker_loader.js

```diff
@@ -15,16 +15,16 @@
 
 'use strict';
 
-//#if !PRODUCTION
-//// Patch importScripts to work around a bug in WebKit and Chrome 48-.
-//// See https://crbug.com/572225 and https://webkit.org/b/153317.
-self.importScripts = (function (importScripts) {
-  return function() {
-    setTimeout(function () {}, 0);
-    return importScripts.apply(this, arguments);
-  };
-})(importScripts);
-//#endif
+if (typeof PDFJSDev === 'undefined' || !PDFJSDev.test('PRODUCTION')) {
+  //// Patch importScripts to work around a bug in WebKit and Chrome 48-.
+  //// See https://crbug.com/572225 and https://webkit.org/b/153317.
```

Comment:
Nit: with the new proprocessor we shouldn't need double comments here, so `////` can be changed to `//`.